### PR TITLE
Added capture list proposal

### PR DESCRIPTION
**Pull request by NSLeoFont**
# About

This PR adds explanation about the use of capture lists instead of weak/strong/self

# Related tasks

https://digitalcrew.teamwork.com/index.cfm#tasks/11781820

### CODE REVIEW

**Comment by josealobato** [comment]
Funny I just read an article about it.

**Review by josealobato — APPROVED** [review]
Love it!

**Review by denisq1991 — APPROVED** [review]
Great addition, thanks @NSLeoFont!

**Review by chillok — APPROVED** [review]

File: README.markdown [hunk 1]
```diff
@@ -898,6 +898,18 @@ resource.request().onComplete { [weak self] response in
 }
 ```
 
+As a *better* alternative to the weak self/strong self scenario, if we just need to capture some variables or objects, instead of addressing the whole self,we can use a capture list with the name of the vars needed.
```
Comment (rkreutz-teamwork) [inline README.markdown]: There is a small typo `self,we` should have a space

**Review by rkreutz-teamwork — APPROVED** [review]

File: README.markdown (hunk 1 above)
Comment (NSLeoFont) [reply README.markdown]: Fixed!

**Pull request merged**